src/c2goto/library/io.c

```diff
@@ -58,11 +68,43 @@ __ESBMC_HIDE:;
   return f;
 }
 
+//  fgets reads a line from the specified stream and stores it
+// into the string pointed to by str. It stops when either (n-1)
+// characters are read, the newline character is read, or the
+// end-of-file is reached, whichever comes first.
+// source: https://www.cplusplus.com/reference/cstdio/fgets/
```

Comment:
It goes saying
> A newline character makes fgets stop reading, but it is considered a valid character by the function and included in the string copied to str.

Reply:
Improved the documentation as suggested.